product-database/main.go

```diff
@@ -0,0 +1,87 @@
+package main
+
+import (
+	"database/sql"
+	"encoding/json"
+	"log"
+	"os"
+)
+
+var Item []struct {
+	ID    int     `json:"id"`
+	Name  string  `json:"name"`
+	Price float64 `json:"price"`
+}
+
+func main() {
+	os.Remove("products.db")
+
+	file, err := os.Create("products.db")
```

Comment:
No need to do this. If the `products.db` file does not exist, the `sql.Open` operation on line 25 will create it.